spyder/config/base.py

```diff
@@ -229,6 +229,7 @@ def get_image_path(name, default="not_found.png"):
 LANGUAGE_CODES = {'en': u'English',
                   'fr': u'Français',
                   'es': u'Español',
+                  'hu': u'Hungarian',

```

Comment:
Our convention is to use the native language name, as you can see with the other languages.

So please make our requested change.